app/src/main/kotlin/com/wire/android/ui/home/conversations/ConversationScreen.kt

```diff
@@ -1048,6 +1058,23 @@ fun MessageList(
                     val showAuthor = rememberShouldShowHeader(index, message, lazyPagingMessages)
                     val useSmallBottomPadding = rememberShouldHaveSmallBottomPadding(index, message, lazyPagingMessages)
 
+                    if (index > 0) {
```

Comment:
Maybe let's add some remember functions to not trigger it every time if it's possible and name variables more understandable `currentGroup` => `currentGroupTime` ?

Reply:
Actually, if we keep a variable on the compose it will get changed when its recomposed (maybe not with remember) but the index its from the lazyList keeps its original reference when changing pages so its currently our source of truth despite being on a different page